test/rootchain/blockSubmissions.js

```diff
@@ -14,48 +13,24 @@ contract('[RootChain] Block Submissions', async (accounts) => {
     });
 
     it("Submit block from authority", async () => {
```

Comment:
Could we get test of batched submit block calls here instead of in `miscellaneuous.js`?